go.mod

```diff
@@ -9,6 +9,6 @@ go 1.13
 require (
 	periph.io/x/conn/v3 v3.6.9
 	periph.io/x/d2xx v0.0.4
-	periph.io/x/devices/v3 v3.6.12
+	periph.io/x/devices/v3 v3.6.13-0.20211210231915-5b35a3fb05bf
```

Comment:
This is necessary because no tagged version contains `videosink` yet.

@maruel, could you perhaps tag a release?

Reply:
Will do.

Reply:
Done, please rebase.

Reply:
I resolved the conflict with a merge in the meantime.

Reply:
It worked, so I rebased and squashed it.